src/rgw/rgw_kafka.h

```diff
@@ -21,28 +21,47 @@ bool init(CephContext* cct);
 // shutdown the kafka manager
 void shutdown();
 
+// key class for the connection list
+struct connection_id_t {
```

Comment:
can you move this to the cc file and just keep forward declaration here?

Reply:
this struct is required in the `rgw_pubsub_push.cc` and i have followed the same logic as we have in `rgw_amqp.h`
any reason to move this to `.cc`?